src/libraries/System.Runtime.Numerics/src/System/Numerics/BigIntegerCalculator.AddSub.cs

```diff
@@ -9,58 +9,131 @@ namespace System.Numerics
 {
     internal static partial class BigIntegerCalculator
     {
+
+#if DEBUG
+        // Mutable for unit testing.
+        private static
+#else
+        private const
+#endif
+        int CopyToThreshold = 8;
```

Comment:
It's fine to use similar things during development, but we can't merge them.

```suggestion
        private const int CopyToThreshold = 8;
```

Reply:
Agree, Plain `private const` looks better